src/pages/using-spark/foundations/css-utilities.mdx

```diff
@@ -104,52 +196,68 @@ supplemental spacing sizes. The class name format is `.sprk-u-<Type><Location>--
 
 #### Examples
 
-<section>
+<section class="sprk-u-mbm">
   <CssUtilityTable group="misc-spacing"/>
 </section>
+<a class="sprk-b-Link sprk-u-Float--right" href="#table-of-contents">
+  Back to Table of Contents
+</a>
 
 ### Color
 
-These utilities control CSS properties related to color and background color.
+These utilities change color and background color.
 
-<section>
+<section class="sprk-u-mbm">
   <CssUtilityTable group="color"/>
 </section>
+<a class="sprk-b-Link sprk-u-Float--right" href="#table-of-contents">
+  Back to Table of Contents
+</a>
 
 ### Height
 
 This utility sets the height of an element to a percentage value.
 
-<section>
+<section class="sprk-u-mbm">
   <CssUtilityTable group="height"/>
 </section>
+<a class="sprk-b-Link sprk-u-Float--right" href="#table-of-contents">
+  Back to Table of Contents
+</a>
 
-## Multi-Property Utilities<a name="multi-prop"></a>
+## Multi-Property Utilities
 
-Multi-Property Utilities combine several CSS properties and values to perform
- common actions. They are not meant to override default styles, but aid in
-  things like accessibility and layout.
+Multi-Property Utilities combine several CSS properties and values to perform common actions. These aid in accessibility and layout, and are not meant to override default styles.
 
 ### Accessibility
 
-These utilities are related to accessibility. These should be used to make the experience more accessible.
+These utilities affect how content will render for screen readers.
```

Comment:
I don't think you should leave this only about screen readers. Might be the case now, but any utility for accessibility would go here whether it is specific to screen readers or not.

Reply:
Good point! Improved~ ✅ ![image](https://user-images.githubusercontent.com/4342363/94475233-c6658500-019c-11eb-8d10-ad878c1aa3cf.png)